test/controllers/web/admin/users_controller_test.rb

```diff
@@ -5,6 +5,7 @@ class Web::Admin::UsersControllerTest < ActionController::TestCase
     admin = create :admin
     sign_in admin
     @user = create :user
+    @exceptions_attributes = ['id', 'created_at', 'updated_at']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.